blazor/diagram/migration-classic-to-native.md

```diff
@@ -0,0 +1,577 @@
+| Set the corner radius property to specify the radius of rounded rectangle. | **Property:** *Nodes[i].Shape.CornerRadius* <br><br>`<SfDiagram Nodes="@NodeCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public ObservableCollection<DiagramNode> NodeCollection = new ObservableCollection<DiagramNode>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`DiagramNode node1 = new DiagramNode()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;`Id = "node",`<br>&nbsp;&nbsp;&nbsp;`OffsetX = 300,`<br>&nbsp;&nbsp;&nbsp;`OffsetY = 288,`<br>&nbsp;&nbsp;&nbsp;`Width = 100,`<br>&nbsp;&nbsp;&nbsp;`Height = 100,`<br>&nbsp;&nbsp;&nbsp;`Shape=new DiagramShape(){Type=Shapes.Basic,BasicShape=BasicShapes.Rectangle, CornerRadius = 10 }`<br>&nbsp;&nbsp;&nbsp;`Annotations = new ObservableCollection<DiagramNodeAnnotation>()`<br>&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;`new DiagramNodeAnnotation()`<br>&nbsp;&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`Content = "node",`<br>&nbsp;&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;`};`<br>&nbsp;&nbsp;`NodeCollection.Add(node1);`<br>&nbsp;`}`<br>`}`| **Property:** *Nodes[i].Shape.CornerRadius* <br><br> Sets the shape as a *BasicShape* <br><br>`<SfDiagramComponent Nodes="@NodeCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public DiagramObjectCollection<Node> NodeCollection = new DiagramObjectCollection<Node>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`Node node1 = new Node()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;`Height = 100,`<br>&nbsp;&nbsp;&nbsp;`Width = 100,`<br>&nbsp;&nbsp;&nbsp;`ID = "node",`<br>&nbsp;&nbsp;&nbsp;`OffsetX = 300,`<br>&nbsp;&nbsp;&nbsp;`OffsetY = 288,`<br>&nbsp;&nbsp;&nbsp;`Shape = new BasicShape() { Type = NodeShapes.Basic, Shape = NodeBasicShapes.Rectangle, CornerRadius = 10 },`<br>&nbsp;&nbsp;&nbsp;`Annotations = new DiagramObjectCollection<ShapeAnnotation>()`<br>&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;`new ShapeAnnotation()`<br>&nbsp;&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`Content="node",`<br>&nbsp;&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;`};`<br>&nbsp;&nbsp;`NodeCollection.Add(node1);`<br>&nbsp;`}`<br>`}`|
```

Comment:
| Sets the corner radius property to specify the radius of the rounded rectangle. | **Property:** *Nodes[i].Shape.CornerRadius* <br><br>`<SfDiagram Nodes="@NodeCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public ObservableCollection<DiagramNode> NodeCollection = new ObservableCollection<DiagramNode>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`DiagramNode node1 = new DiagramNode()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;`Id = "node",`<br>&nbsp;&nbsp;&nbsp;`OffsetX = 300,`<br>&nbsp;&nbsp;&nbsp;`OffsetY = 288,`<br>&nbsp;&nbsp;&nbsp;`Width = 100,`<br>&nbsp;&nbsp;&nbsp;`Height = 100,`<br>&nbsp;&nbsp;&nbsp;`Shape=new DiagramShape(){Type=Shapes.Basic,BasicShape=BasicShapes.Rectangle, CornerRadius = 10 }`<br>&nbsp;&nbsp;&nbsp;`Annotations = new ObservableCollection<DiagramNodeAnnotation>()`<br>&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;`new DiagramNodeAnnotation()`<br>&nbsp;&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`Content = "node",`<br>&nbsp;&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;`};`<br>&nbsp;&nbsp;`NodeCollection.Add(node1);`<br>&nbsp;`}`<br>`}`| **Property:** *Nodes[i].Shape.CornerRadius* <br><br> Sets the shape as a *BasicShape* <br><br>`<SfDiagramComponent Nodes="@NodeCollection" />`<br>`@code`<br>`{`<br>&nbsp;`public DiagramObjectCollection<Node> NodeCollection = new DiagramObjectCollection<Node>();`<br>&nbsp;`protected override void OnInitialized()`<br>&nbsp;`{`<br>&nbsp;&nbsp;`Node node1 = new Node()`<br>&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;`Height = 100,`<br>&nbsp;&nbsp;&nbsp;`Width = 100,`<br>&nbsp;&nbsp;&nbsp;`ID = "node",`<br>&nbsp;&nbsp;&nbsp;`OffsetX = 300,`<br>&nbsp;&nbsp;&nbsp;`OffsetY = 288,`<br>&nbsp;&nbsp;&nbsp;`Shape = new BasicShape() { Type = NodeShapes.Basic, Shape = NodeBasicShapes.Rectangle, CornerRadius = 10 },`<br>&nbsp;&nbsp;&nbsp;`Annotations = new DiagramObjectCollection<ShapeAnnotation>()`<br>&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;`new ShapeAnnotation()`<br>&nbsp;&nbsp;&nbsp;&nbsp;`{`<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`Content="node",`<br>&nbsp;&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;&nbsp;`}`<br>&nbsp;&nbsp;`};`<br>&nbsp;&nbsp;`NodeCollection.Add(node1);`<br>&nbsp;`}`<br>`}`|